tex/alg-NT/ramification.tex

```diff
@@ -262,22 +262,27 @@ \section{(Optional) Decomposition and inertia groups}
 	How are $\Gal\left( (\OO_K/\kp) / \FF_p \right)$
 	and $\Gal(K/\QQ)$ related?
 \end{quote}
-Absurdly enough, there is an explicit answer:
-\textbf{it's just the stabilizer of $\kp$, at least when
-$p$ is unramified}.
+
```

Comment:
("It's natural to consider the stabilizer, not absurd!")

In fact you somewhat derived the result right below (explain how we should consider the stabilizer), I mostly just reorder it to motivate it better.